examples/UAVCAN-Blink/UAVCAN-Blink.ino

```diff
@@ -0,0 +1,163 @@
+/*
+ * This example creates a UAVCAN node. The builtin LED can be switched on and off using UAVCAN.
+ * It also shows periodic transmission of a UAVCAN heartbeat message via CAN.
+ *
+ * switch built in LED on with
+ *   pyuavcan pub 1620.uavcan.primitive.scalar.Bit.1.0 {'value: true'} --tr='CAN(can.media.socketcan.SocketCANMedia("can0",8),59)'
```

Comment:
I'm a bit surprised that this line actually works, since the node if for the UAVCAN is actually 13. Should this comment not rather be:
```bash
pyuavcan pub 1620.uavcan.primitive.scalar.Bit.1.0 {'value: true'} --tr='CAN(can.media.socketcan.SocketCANMedia("can0",8),13)'
```

Reply:
It seems like there is a missunderstanding of the uavcan concept on your side. ;-)
13 is the Node-ID of the Arduino board. 59 is the Node-ID of the computer publishing the subject. uavcan relies only on the Subject-ID. So any Node-ID publishing to the Subject-ID 1620 will be accepted.
Actually the Node-ID of the computer has to be different of "13" since Node-IDs have to be unique in the whole system.

Reply:
You caught me there 🙇 . Please consider my comment revoked.

Reply:
```suggestion
 *   pyuavcan pub 1620.uavcan.primitive.scalar.Bit.1.0 'value: true' --tr='CAN(can.media.socketcan.SocketCANMedia("can0",8),59)'
```

Braces around the string literal are meaningless and may break the command for some shells. You mean either `'value: true'` or `'{value: true}'`. See YAML docs at https://yaml.org/

The same applies to the command below this one.